### PR TITLE
Migrate user guide from Address Book 3 to Mark

### PR DESCRIPTION
**Pull request by openorclose**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
+ . Type the command in the command box and press Enter to execute it.
+
+* e.g. typing help and pressing Enter will open the help window.
+. Some example commands you can try:
 
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
-* *`exit`* : exits the app
+* *list* : lists all contacts
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]:
```suggestion
* *list* : lists all bookmarks
```

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
 *Command Format*
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-====
+* Words in UPPER_CASE are the parameters to be supplied by the user e.g.
+in add u/URL, URL is a parameter which can be used as add
+n/luminus.nus.edu.sg.
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]:
```suggestion
u/luminus.nus.edu.sg.
```

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
+ * Items in square brackets are optional e.g u/URL [t/TAG] can be used as
+n/luminus.nus.edu.sg t/NUS or as n/luminus.nus.edu.sg.
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]:
```suggestion
u/luminus.nus.edu.sg t/NUS or as u/luminus.nus.edu.sg.
```

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
+Deletes the specified bookmark from the bookmark manager.
 
-****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
-****
+Format/ *delete INDEX*
```
Comment (dorcastan) [inline docs/UserGuide.adoc]: Format: *delete INDEX*

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -18,160 +18,598 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
+Goes to the Online tab which is a web view
+
+* *tab 3* +
+Goes to the Offline tab which shows the offline copy of bookmarks
 
 == FAQ
 
-*Q*: How do I transfer my data to another Computer? +
-*A*: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Address Book folder.
+*What are Tabs?*
```
Comment (dorcastan) [inline docs/UserGuide.adoc]: We should probably make the FAQs headings as well, so we can link to them directly.

**Comment by Na-Nazhou** [comment]
#9

**Pull request merged**